avm/utilities/pipelines/staticValidation/psrule/ps-rule.yaml

```diff
@@ -80,3 +80,5 @@ rule:
     - Azure.KeyVault.PurgeProtect
     - Azure.VM.UseHybridUseBenefit
     - Azure.AppConfig.PurgeProtect
+    - Azure.APIM.MultiRegion # Team agreed this is too expensive for most use cases and is safe to ignore.
```

Comment:
Line 83 seemed to have been slipped in here. I think this value has since been moved to the min suppression file

Reply:
Fixed.